qa/tasks/vstart_runner.py

```diff
@@ -935,7 +935,7 @@ def run_ceph_w(self, watch_channel=None):
         if watch_channel is not None:
             args.append("--watch-channel")
             args.append(watch_channel)
-        proc = self.controller.run(args=args, wait=False, stdout=BytesIO())
+        proc = self.controller.run(args=args, wait=False, stdout=StringIO())
```

Comment:
ahh, should add `from six import StringIO` at the top of this file.

Reply:
@tchaikov is it `six` or `io`, just asking because for `BytesIO` it's `from io import BytesIO` in the vstart_runner.py version of [octopus](https://github.com/ceph/ceph/blob/octopus/qa/tasks/vstart_runner.py#L33).

Reply:
`six`.

because before using python3 for driving the octopus tests, we should be python2 compatible, but `io.StringIO` in python2 expects **unicode**, while `StringIO.StringIO` in python2 expects str. i don't want to investigate if any of vstart_runner's consumer would be annoyed at seeing unicode on Python3, so a safe bet would be `six.StringIO` which is an alias of `StringIO.StringIO` on python2 and `io.StringIO` on python3. but if you are sure about this, `io.StringIO` is the way to go.

but we don't have this issue when it comes to `BytesIO` .

Reply:
Thanks for the detailed explanation, @tchaikov!

Reply:
As discussed offline with @tchaikov I added the missing import `from six import StringIO` - I added it to the backport commit which uses `StringIO` with an explanation why this change can not be cherry-picked from master.